.gitlab-ci.yml

```diff
@@ -316,3 +316,31 @@ puppet-release:
   artifacts:
     paths:
       - deployments/puppet/pkg/*.tar.gz
+
+cve-scan:
+  image: ubuntu:20.04
+  stage: test
```

Comment:
I'm not an expert in Gitlab config but shouldn't the `test` stage be added to `stages:` list? Same for the next job...

Reply:
Yes! I missed that. thank you. Should we call it `test` or `scan` sounds better?

Reply:
`test` is a good option. It should probably go in between `build` and `sign`

Reply:
ok, thank you